source/Core/Membership/Memberships/Temporary/Add/AddTemporaryMembershipBO.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace NEBA.Membership.Memberships.Temporary.Add
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class BusinessLogic : Memberships.Add.BusinessLogic<Models.TemporaryMembership>
+    {
+        /// <summary>
+        /// Unit Test Constructor

```

Comment:
remove comment (not unit test constructor)
```suggestion
        ///

```